### ISSUE TITLE
Can't get it working. Console shows an Error Message.

### ISSUE THREAD

**Issue by tobiashochguertel** [SnoringFrog/GitlabIntegrate#8]
**Sublime 3 Console Error Message Output:**
```
reloading plugin GitlabIntegrate.gitlab_integrate
Traceback (most recent call last):
  File "D:\Applications\Sublime Text 3\sublime_plugin.py", line 74, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 584, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1022, in load_module
  File "<frozen importlib._bootstrap>", line 1003, in load_module
  File "<frozen importlib._bootstrap>", line 560, in module_for_loader_wrapper
  File "<frozen importlib._bootstrap>", line 868, in _load_module
  File "<frozen importlib._bootstrap>", line 313, in _call_with_frames_removed
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab_integrate.py", line 9, in <module>
    from . import gitlab
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\__init__.py", line 7, in <module>
    from . import requests
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\requests\__init__.py", line 58, in <module>
    from . import utils
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\requests\utils.py", line 23, in <module>
    from .compat import parse_http_list as _parse_list_header
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\requests\compat.py", line 7, in <module>
    from .packages import charade as chardet
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\requests\packages\__init__.py", line 3, in <module>
    from . import urllib3
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\requests\packages\urllib3\__init__.py", line 16, in <module>
    from .connectionpool import (
  File "C:\Users\hochguertelto\AppData\Roaming\Sublime Text 3\Packages\GitlabIntegrate\gitlab\requests\packages\urllib3\connectionpool.py", line 40
    except (ImportError, AttributeError): # Platform-specific: No SSL.
         ^
SyntaxError: invalid syntax
```

**User Configuration:**
```
{
	"display_intro":true,
	"edit_issue_in_tab_name": "[GLI]: Editing Issue",
	"hide_closed_issues":false,
	"output_prefix":"[GLI]:",
	"project_host":"http://gitlab.hochguertel.biz",
	"project_id": 0,
	"user_token":"aaBBccDD01010101"	
}
```

*Token changed for this case, I can provide an Test Account on gitlab.hochguertel.biz, please speak to me.*

**Enviroment Information:**

GitLab 7.9.1 6f6c5f1
Sublime Text 3 Stable Channel, Build 3083
Gitlab​Integrate 1.4.11, Package Control
Windows 7 Ultimate SP 1 (64-bit)

**Comment by SnoringFrog** [comment]
Not sure why this problem randomly surfaced now, but it should be fixed with the most recent update. Make sure you have version 1.5.0 and let me know if the problem is not fixed then.

**Comment by SnoringFrog** [comment]
This should be fixed in version 1.5.1. Make sure you've updated and let me know if that did/didn't fix the problem for you.

**Comment by tobiashochguertel** [comment]
@SnoringFrog updated to version 1.5.1 via package control. Now it works. Thanks. :+1:
